h/api/models/test/document_test.py

```diff
@@ -434,6 +435,202 @@ def merge_data(self, request):
         return (master, duplicate)
 
 
+class TestUpdateDocumentMetadata(object):
+
+    def test_it_calls_find_or_create_by_uris(self,
```

Comment:
You don't need to change this now, but in general it's really helpful to me when reading the tests if they're named in such a way that their purpose is clear. Test names that express motivation, not implementation, are very helpful:

```
test_retrieves_document_using_uri
test_finds_existing_documents
```

are probably better names than

```
test_it_calls_some_function
```

if for no other reason than a code refactoring that results in `some_function` being renamed will result in test names that make no sense. More broadly, good test names (or, failing that, good test docstrings) really help when trying to determine if a test failure is because the tests are broken or the code is broken.